include/effects/strip/faneffects.h

```diff
@@ -450,8 +450,6 @@ class FanBeatEffect : public LEDStripEffect
   }
 };
 
-extern void ShowTM1814();
-

```

Comment:
I removed all references to ShowTM1814(). It didn't seem to be used anywhere, and CI is happy without it.